gpt4all-backend/llmodel_c.cpp

```diff
@@ -51,6 +52,20 @@ void llmodel_llama_destroy(llmodel_model llama)
     delete wrapper;
 }
 
+llmodel_model llmodel_replit_create()
+{
+    LLModelWrapper *wrapper = new LLModelWrapper;
```

Comment:
```c++
auto *wrapper = new LLModelWrapper;
```

(Yes, I know this is different from the existing functions. They aren't perfect either.

Reply:
This proposed change adds nothing. I'd prefer reviews focused on the patch and not coding style. The only thing I ask for coding style is that you try and match what is already used in the file and/or the project as a whole.